cmd/prometheus/main.go

```diff
@@ -278,6 +278,9 @@ func (c *flagConfig) setFeatureListOptions(logger *slog.Logger) error {
 			case "otlp-deltatocumulative":
 				c.web.ConvertOTLPDelta = true
 				logger.Info("Converting delta OTLP metrics to cumulative")
+			case "use-direct-io":
```

Comment:
maybe a more generic `use-uncached-io` so we can seamlessly switch the underlying feature, see https://github.com/prometheus/proposals/pull/45#discussion_r1982030536